### ISSUE TITLE
[Bug]: Album cover from previous song is shown

### ISSUE THREAD

**Issue by flippflapp** [enricocid/Music-Player-GO#477]
### What happened?

Sometimes, when skipping to the next song, the album cover of the previous song is still shown both on the lock screen and in the notification centre (it is correctly shown in the main view). It doesn't always happen, I can't quite say under which conditions it'll happen. This did happen before the current MPGO's version.

### Version

4.4.22

### Device and Android version

Device: Pixel 4a
OS: Android 13
ROM or mods: GrapheneOS Build TP1A.221005.002.2022100300

### More info

Example image:
<img height="450" src="https://user-images.githubusercontent.com/98783999/194945590-08e18fac-4ba8-473e-ab64-f73ce09f657d.png"> 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by OxygenCobalt** [comment]
Yeah, I encountered that issue @enricocid. It used to be on specific Android 12 models, but now it's ubiquitous on Android 13. You want to sync up your cover loading so that you update your notification and media session at the same time. Otherwise one will override the other and create an issue like this.

**Comment by enricocid** [comment]
Thanks @OxygenCobalt gonna check auxio then 😆

**Comment by enricocid** [comment]
Test apk.
:)
https://drive.google.com/file/d/1Z6szw3NisweIpeQ6gWDhluYVHQDZmLxS/view?usp=sharing

**Comment by flippflapp** [comment]
At first it seemed this issue was resolved, no delayed covers. But then I had a couple of songs with cover for which the cover wasn't shown at all :(

(Didn't resolve #365)

**Comment by enricocid** [comment]
> At first it seemed this issue was resolved, no delayed covers. But then I had a couple of songs with cover for which the cover wasn't shown at all :( I tried around a bit and if I encounter a song with cover for which the cover is not shown, but I go back to the previous song and then forward again to the song in question, the cover _is_ shown.
> 
> (Didn't resolve #365)

Yeah, there was an error in the code.

Try this:
https://drive.google.com/file/d/1McG_kf1DcFTBoWVRp51HMR78_YzRTc7S/view?usp=sharing

now the notification is posted once the bitmap is loaded. :)

**Comment by flippflapp** [comment]
Yes, this seems to resolve this issue! :)
